### ISSUE TITLE
When series_id is supplied to read_abs(), it fetches the most-recently-released table containing the ID; this is not always desirable

### ISSUE THREAD

**Issue by ch3lle82** [MattCowgill/readabs#71]
Hello, 
I am trying to download a number of ABS labour force series using the read_abs() function and by series ID. However, I have noticed that the series that is read in is truncated in some instances. For example, when the ABS series A84423032W, A84423060F and A84423046K are imported, the series starts from January 1991, but in the ABS Preliminary Labour Force Survey publication (ABS Cat. No. 6202.0 Table 1), these series start from February 1978. Is there a way to import the longest series available and/or import data from a certain date using the read_abs() by series ID function?
Thanks
Michelle

**Comment by MattCowgill** [comment]
Hi,
Some time series IDs are part of multiple ABS statistical releases.
For example, the ID `A84423032W` that you provided is part of ABS 6202.0, table 1, which goes back to 1978.

However, ID `A84423032W` is also part of ABS 6291.0.55.001, Table 14b, which goes back to 1991.

Where a time series ID is part of multiple releases, `read_abs()` is currently configured to get the most recent release. Given that 6291.0.55.001 was released more recently than 6202.0, it returns the series that begins in 1991.

I agree this result is undesirable. It is unintended. I will amend the package so that, where there a single time series ID is found in multiple releases, `read_abs()` returns the one with the earliest start date (assuming they have equal end dates). 

In the meantime, I suggest you read by table, rather than series ID. For example, this code downloads 6202.0 table 1 and filters so that it contains the three series IDs you're looking for: 

``` r

library(readabs)
#> Environment variable 'R_READABS_PATH' is unset. Downloaded files will be saved in a temporary directory.
#> You can set 'R_READABS_PATH' at any time. To set it for the rest of this session, use
#>  Sys.setenv(R_READABS_PATH = <path>)
library(tidyverse)

x <- read_abs("6202.0", 1) 
#> Finding filenames for tables corresponding to ABS catalogue 6202.0
#> Attempting to download files from catalogue 6202.0, Labour Force, Australia
#> Extracting data from downloaded spreadsheets
#> Tidying data from imported ABS spreadsheets

x <- x %>%
  filter(series_id %in% c("A84423032W",
                          "A84423060F",
                          "A84423046K")) 
unique(x$series)
#> [1] "Unemployed total ;  Persons ;"   "Unemployed total ;  > Males ;"  
#> [3] "Unemployed total ;  > Females ;"
```

<sup>Created on 2020-05-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by ch3lle82** [comment]
Great, thanks so much for your quick response!
